transport.go

```diff
@@ -157,21 +167,14 @@ func getRequestFromEvent(event *Event, dsn *Dsn) (r *http.Request, err error) {
 	if body == nil {
 		return nil, errors.New("event could not be marshaled")
 	}
-	if event.Type == transactionType {
-		b, err := transactionEnvelopeFromBody(event, dsn, time.Now(), body)
-		if err != nil {
-			return nil, err
-		}
-		return http.NewRequest(
-			http.MethodPost,
-			dsn.EnvelopeAPIURL().String(),
-			b,
-		)
+	b, err := envelopeFromBody(event, dsn, time.Now(), body)
```

Comment:
Let's rename `b` to something more human-friendly

Reply:
Don't make the Go gods angry 😬
Renamed to `envelope`.